DQMOffline/HLTScouting/python/HLTScoutingDqmOffline_cff.py

```diff
@@ -11,11 +11,18 @@
 from HLTriggerOffline.Scouting.ScoutingMuonTagProbeAnalyzer_cfi import *
 from HLTriggerOffline.Scouting.ScoutingMuonMonitoring_Client_cff import *
 
+
 from HLTriggerOffline.Scouting.HLTScoutingEGammaDqmOffline_cff import *
+from HLTriggerOffline.Scouting.ScoutingCollectionMonitor_cfi import *
 
 hltScoutingMuonDqmOffline = cms.Sequence(scoutingMonitoringTagProbeMuonNoVtx
                                          * scoutingMonitoringTagProbeMuonVtx                                                         
                                          * scoutingMonitoringTriggerMuon                                                              
 ) 
 
-hltScoutingDqmOffline = cms.Sequence(hltScoutingMuonDqmOffline + hltScoutingEGammaDqmOffline)
+hltScoutingJetDqmOffline = cms.Sequence(jetMETDQMOfflineSourceScouting)
```

Comment:
I think this PR fails because https://github.com/cms-sw/cmssw/pull/47328/ was not backported.